apps/emqx/src/emqx_schema.erl

```diff
@@ -1678,12 +1678,12 @@ fields("durable_sessions") ->
                     desc => ?DESC(session_ds_idle_poll_interval)
                 }
             )},
-        {"last_alive_update_interval",
+        {"heartbeat_interval",
             sc(
                 timeout_duration(),
                 #{
                     default => <<"5000ms">>,
-                    desc => ?DESC(session_ds_last_alive_update_interval)
+                    desc => ?DESC(session_ds_heartbeat_interval)
```

Comment:
I wonder why spellcheck does not say anything about this being undefined in the respective `emqx_schema.hocon`.

Reply:
I think this is because this parameter is hidden. 
I'll make a change to unhide later.